homeassistant/components/yeelight/light.py

```diff
@@ -651,7 +651,7 @@ def set_default(self) -> None:
     def set_flash(self, flash) -> None:
         """Activate flash."""
         if flash:
-            if self._bulb.last_properties["color_mode"] != 1:
+            if self._bulb.last_properties["color_mode"] != "1":
```

Comment:
Isn't it saver to cast here?

```suggestion
            if int(self._bulb.last_properties["color_mode"]) != 1:
```

Reply:
Yes I guess it's better to do the cast. I'll change that.